src/compiler/diagnosticMessages.json

```diff
@@ -1007,7 +1007,6 @@
         "category": "Error",
         "code": 1349
     },
-

```

Comment:
I think these lines are just better for navigating the file? Like move between empty lines so you can navigate the file faster.